ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml

```diff
@@ -172,6 +172,28 @@
                     DISTROS=centos8
                     FLAVOR=crimson
                     ARCHS=x86_64
+      # Build jaegertracing branch on needed env,  don't waste resources on the default one.
+      # Useful for testing specific builds failure
+      # default: focal
+      # crimson: centos8
+      - conditional-step:
+          condition-kind: regex-match
+          regex: .*jaeger-only.*
+          label: '${GIT_BRANCH}'
+          on-evaluation-failure: dont-run
+          steps:
+            - shell:
+                !include-raw:
+                - ../../../scripts/build_utils.sh
+                - ../../build/notify
+            - trigger-builds:
+                - project: 'ceph-dev-new'
+                  predefined-parameters: |
+                    BRANCH=${GIT_BRANCH}
+                    FORCE=True
+                    DISTROS=centos8 focal
```

Comment:
what you need is to add a flavor of "jaeger" to `ceph-dev-new/config/definitions/ceph-dev-new.yml`, so it's another choice of `FLAVOR` parameter. and this parameter can be consumed by the function of `ceph_build_args_from_flavor()` in the shell script of `ceph-dev-new-setup/build/build`

Reply:
or you can define another parameter instead of reusing to FLAVOR, but the idea is 

1. do not use `FLAVOR=crimson` for buliding jaeger, as they are not related.
2. use this machinery to pass a message to script used by the downstream project of `ceph-dev-new`  that we want build with jaeger.

Reply:
done, jaeger could be tested both by specifying in branch name, as well as passing FLAVOR=jaeger now